src/app/components/outcomes-model/outcomes-model.component.css

```
@@ -10,14 +10,13 @@
   --ink: var(--oc-ink);
   --muted: var(--oc-muted);
   --line: var(--oc-line);
-  background: var(--oc-transparent);
+  background: var(--oc-white);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)